velox/common/base/Counters.cpp

```diff
@@ -236,6 +241,11 @@ void registerVeloxMetrics() {
   DEFINE_METRIC(
       kMetricSsdCacheReadCheckpointErrors, facebook::velox::StatType::SUM);
 
+  // Total number of SSD cache reads without checksum verification due to the
```

Comment:
// Total number of SSD cache reads without checksum verification due to mismatch in SSD cache request size.